styles.css

```diff
@@ -1 +1,77 @@
-/* this will be the styles file */
+* {
+  background-color: black;
+}
+body {
+  margin: 0px 160px;
+}
+.logo {
+  display: flex;
+  color: #fff;
+  justify-self: start;
+  align-items: center;
+}
+nav {
+  color: #fff;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  gap: 48px;
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 26px;
+}
+.support-button {
+  background-color: #f4511f;
+  color: #fff;
+  border-radius: 24px;
+  padding: 9px 24px;
+  font-size: 16px;
+  font-weight: 400;
+  width: 112px;
+  height: 41px;
+}
+nav {
+  display: flex;
+  display: inline-flex;
+  justify-content: space-evenly;
+}
```

Comment:
same selector used above, can we combine these?

Reply:
done